interface.js

```diff
@@ -0,0 +1,66 @@
+function getPostData() {
+    return fetch(`https://content.guardianapis.com/search?section=politics&api-key=9145e040-083e-4e42-9b33-9f8224375a76&show-fields=all`)
```

Comment:
nice to see the use of the API without using the Makers API to access the data. Good to see how this would work.